srv/modules/runners/push.py

```diff
@@ -137,9 +137,14 @@ def _custom(self, custom):
             if not os.path.isfile(custom):
                 log.info("Writing {}".format(custom))
                 with open(custom, "w") as yml:
-                    yml.write("# Customizations for {}\n".format(custom))
-    
-    
+                    custom_split = custom.split("stack")
+                    custom_for = "{}{}{}".format(
+                            custom_split[0],
+                            "stack/default",
+                            custom_split[1])
+                    yml.write("# Customizations for {}\n".format(custom_for))
```

Comment:
Where is pathname from?
In any case imho right now it sounds like this file overwrites the content in the default tree. Maybe we should add two comments? Something like:
`# This file will overwrite configuration made in /srv/pillar/ceph/stack/default/global.yml when put in /srv/pillar/ceph/stack/global.yml`